packages/wrangler/src/api/startDevWorker/ProxyController.ts

```diff
@@ -161,6 +161,7 @@ export class ProxyController extends Controller<ProxyControllerEventMap> {
 					logger.loggerLevel === "debug" ? "wrangler-ProxyWorker" : "wrangler",
 			}),
 			handleRuntimeStdio,
+			liveReload: false,
```

Comment:
Is the default true? If so, that was an interesting decision on the part of Miniflare...

Reply:
Seems so. Only affects responses with html content type but yeah agree. startWorker's default is false